buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -21,4 +21,5 @@ object Versions {
     const val kotlinJsWrappersSuffix = "-pre.148-kotlin-1.4.30"
     const val kotlinReact = "$react$kotlinJsWrappersSuffix"
     const val jgit = "5.10.0.202012080955-r"
+    const val ktReflect = "1.4.31"
```

Comment:
I suppose it shares the version with kotlin? Why separate variable?

Reply:
Yes, fixed